dyno-queues-redis/src/main/resources/demo.properties

```diff
@@ -0,0 +1,32 @@
+####
+## Properties to initialize the demo app
+#
+LOCAL_DATACENTER=us-east-1
+LOCAL_RACK=us-east-1c
+NETFLIX_STACK=dyno_demo
+#EC2_AVAILABILITY_ZONE=us-east-1
+
+dyno.demo.lbStrategy=TokenAware
+
+netflix.appinfo.name=dyno_demo
+netflix.environment=test
+netflix.appinfo.metadata.enableRoute53=false
+netflix.discovery.registration.enabled=false
+netflix.appinfo.validateInstanceId=false
+
+dyno.demo.retryPolicy=RetryNTimes:2
+dyno.demo.port=8102
+dyno.demo.discovery.prod=discoveryreadonly.%s.dynprod.netflix.net:7001/v2/apps
+dyno.demo.discovery.test=discoveryreadonly.%s.dyntest.netflix.net:7001/v2/apps
+
+##
+# These properties apply to runScanTest(). Note that .prefix will apply only if .populateKeys is true. To query for a
```

Comment:
Are the "scan" properties still required?

Reply:
i took these directly from dyno. you're right. have pulled in some unnecessary ones. will fix.